HW_3/hw_3_3.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/*
+Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются
+*/
+
+function concatProgram (n) {
+    let result;
+    if (n >= 1 && n <= 9 && String(n/1).length === 1) {
```

Comment:
This validation will look better and more readable)
```suggestion
    if (n && typeof n === 'number' && n >= 1 && n <= 9) {
```